package.json

```diff
@@ -28,6 +28,7 @@
     "lint": "node_modules/.bin/eslint zapier.js src",
     "lint-snippets": "node_modules/.bin/eslint snippets --rule 'no-unused-vars: 0' --ignore-pattern snippets/next.js",
     "test": "npm run build && NODE_ENV=test node_modules/mocha/bin/mocha -t 30000 --recursive lib/tests && npm run lint && npm run lint-snippets",
+    "smoke-test": "npm run build && NODE_ENV=test node_modules/mocha/bin/mocha -t 30000 --recursive lib/smoke-tests",
```

Comment:
`node_modules/mocha/bin/mocha` can just be `mocha` (same with above line)

Reply:
also, might make a `pretest` script that runs build and run it manually from here (it'll get picked up before `test` automatically)

Reply:
Nice suggestion! Fixed in 00f2354.